scalameter-core/src/main/scala/org/scalameter/Measurer.scala

```diff
@@ -251,19 +279,20 @@ object Measurer {
       }
 
       var best = results
-      while (outlierExists(results) && retleft > 0) {
-        val prefixlen = suffixLength(results.reverse)
-        val suffixlen = suffixLength(results)
-        val formatted = results.map(t => f"$t%.3f")
+      while (outlierExists(results.map(_.value.toDouble())) && retleft > 0) {
+        val prefixlen = suffixLength(results.reverse.map(_.value.toDouble()))
+        val suffixlen = suffixLength(results.map(_.value.toDouble()))
+        val formatted = results.map(t => f"${t.value.toDouble()}%.3f")
```

Comment:
`Numeric[Double].toDouble(d)` returns value itself and also won't f"num$.3f" formatter guarantee us that any `Double` will be formatted to max three digits after decimal point? Also note, that such formatter requires `Double` values.